pkg/service/v1alpha1/controller.go

```diff
@@ -69,7 +69,6 @@ func (cs *controller) CreateVolume(
 
 	volName := req.GetName()
 	size := req.GetCapacityRange().RequiredBytes
-	configClass := req.GetParameters()["configClass"]
```

Comment:
does this impacts maya code?

Reply:
this will not impact maya code, in maya this variable doesn't need to be necessarily set.